packages/ckeditor5-font/src/fontfamily/fontfamilyui.ts

```diff
@@ -73,6 +83,46 @@ export default class FontFamilyUI extends Plugin {
 
 			return dropdownView;
 		} );
+
+		editor.ui.componentFactory.add( `menuBar:${ FONT_FAMILY }`, locale => {
+			const menuView = new MenuBarMenuView( locale );
+
+			menuView.buttonView.set( {
+				label: accessibleLabel,
+				icon: fontFamilyIcon
+			} );
+
+			const listView = new MenuBarMenuListView( locale );
+
+			for ( const definition of listOptions ) {
+				const listItemView = new MenuBarMenuListItemView( locale, menuView );
+				const buttonView = new MenuBarMenuListItemButtonView( locale );
+
+				buttonView.extendTemplate( {
+					attributes: {
+						'aria-checked': buttonView.bindTemplate.to( 'isOn' )
```

Comment:
I think at this point we really need `ButtonView#ariaChecked` and:
* An update here,
* An update in `FontSizeUI`,
* An update in `HeadingUI` (it's missing),
* An update in dropdown utils (`bindViewCollectionItemsToDefinitions()`).